spanner-cassandra-launcher/src/main/java/com/google/cloud/spanner/adapter/SpannerCassandraLauncher.java

```diff
@@ -45,6 +49,7 @@
  * -Dhost=127.0.0.1 \
  * -Dport=9042 \
  * -DnumGrpcChannels=4 \
+ * -DmaxCommitDelayMillis=100 \
```

Comment:
I would change this to 5 (`-DmaxCommitDelayMillis=5`)

Reply:
done